packages/lodestar/test/utils/mocks/chain/chain.ts

```diff
@@ -36,7 +35,7 @@ export class MockBeaconChain implements IBeaconChain {
   readonly genesisTime: Number64;
   readonly genesisValidatorsRoot: Root;
   readonly bls: IBlsVerifier;
-  forkChoice!: IForkChoice;
+  forkChoice: IForkChoice;
```

Comment:
Remove evil null assertion here. ForkChoice has to be defined in a mock BeaconChain